backend/repository/data/client/contacts_client_update.json

```diff
@@ -4,7 +4,7 @@
     "type": "http",
     "thriftFile": "clients/contacts/contacts.thrift",
     "exposedMethods": {
-        "SaveContacts": "Contacts::saveContacts"
+        "saveContacts": "Contacts::saveContacts"
```

Comment:
the method name in the thrift service is lower case so i make it the same as the service name. anyway I will make it the upper case then.